backends/arm/test/conftest.py

```diff
@@ -43,7 +43,7 @@ def pytest_configure(config):
             )
         # Only enable if we also have the TOSA reference model available.
         pytest._test_options["corstone_fvp"] = True  # type: ignore[attr-defined]
-
+    pytest._test_options["llama_inputs"] = config.option.llama_inputs  # type: ignore[attr-defined]
```

Comment:
Not sure if this is general enough to go through the pytest test_options. Any other options?

Reply:
I don't know of an alternative at the moment, do you have any suggestions?

Reply:
Is --model-inputs better perhaps?